### PR TITLE
VueJS cards

### PR DESCRIPTION
**Pull request by dreyhiflden**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -19,5 +19,17 @@ new Vue({
     remove() {
       this.cards.pop();
     },
+
+    removeThis(card) {
+      this.cards.splice(this.cards.indexOf(card), 1)
```
Comment (carrot-e) [inline index.js]: `;`

File: index.js [hunk 2]
```diff
@@ -19,5 +19,17 @@ new Vue({
...
+    },
+
+    setHide(card) {
+      this.$set(card, 'hide', true);
```
Comment (carrot-e) [inline index.js]: Please remind me to discuss this solution at the lesson (if I forget).

File: index.js (hunk 1 above)
Comment (dreyhiflden) [reply index.js]: Fixed

File: index.js (hunk 2 above)
Comment (dreyhiflden) [reply index.js]: Deal

**Pull request merged**